src/lib/proof-system/proof.ts

```diff
@@ -92,6 +92,13 @@ class ProofBase<Input = any, Output = any> {
   publicFields() {
     return (this.constructor as typeof ProofBase).publicFields(this);
   }
+
+  static _proofFromBase64(proofString: string, maxProofsVerified: 0 | 1 | 2) {
+    return Pickles.proofOfBase64(proofString, maxProofsVerified)[1];
```

Comment:
```suggestion
    if (!areBindingsInitialized) {
      throw Error(
        'Pickles bindings must be initialized before calling this method'
      );
    }
    return Pickles.proofOfBase64(proofString, maxProofsVerified)[1];
```

What if we did something like this, coupled with update the export in snarky.js

```
export {
  Snarky,
  Ledger,
  Pickles,
  Test,
  withThreadPool,
  wasm,
  initializeBindings,
  isInitialized as areBindingsInitialized, <-- add this
};
```

That way no one accidentally gets into trouble with this.